assemblyscript.html.markdown

```diff
@@ -0,0 +1,202 @@
+---
+language: Assemblyscript
+contributors:
+    - ["Philippe Vlérick", "https://github.com/pvlerick"]
+    - ["Steve Huguenin-Elie", "https://github.com/StEvUgnIn"]
+    - ["Sebastian Speitel", "https://github.com/SebastianSpeitel"]
+    - ["Max Graey", "https://github.com/MaxGraey"]
+filename: learnassemblyscript.ts
```

Comment:
Does AssemblyScript not use a different file ext?

Reply:
No.